packages/f-searchbox/src/services/location/google/tests/index.test.js

```
@@ -1,7 +1,7 @@
 import LocationGoogle from '..';
 import { loadGoogleMapsMocks } from '../../../../utils/tests/testHelpers';
 
-describe('`LocationGoogle`', () => {
+xdescribe('`LocationGoogle`', () => {
```

Will look at this in another PR. It's causing some silent errors.

---

Fixed: https://github.com/justeat/fozzie-components/pull/545